arlas-tagger-core/src/main/java/io/arlas/tagger/model/TaggingStatus.java

```diff
@@ -57,4 +57,8 @@ public synchronized UpdateResponse updateStatus(TagRefRequest tagRequest, Update
         statusMap.put(tagRequest.id, updateResponse, statusTimeout);
         return updateResponse;
     }
+
+    public void reset() {
+        statusMap = new SelfExpiringHashMap<>();
```

Comment:
Is there no issue by loosing the reference to statusMap (i.a. is it safer to empty the map instead)?

Reply:
My comment isn't relevant as statusMap isn't exposed. 
However as `reset` isn't synchronized, are you sure there won't be troubles if one thread calls `updateStatus` while another calls `reset` or `initStatus`?

Reply:
Good point. I've changed my code accordingly.